code/modules/mob/living/silicon/silicon.dm

```diff
@@ -51,8 +51,12 @@
 
 	var/obj/item/modular_computer/pda/silicon/modularInterface
 
+	voice_filter = "afftfilt=real='hypot(re,im)*sin(0)':imag='hypot(re,im)*cos(0)':win_size=512:overlap=1,rubberband=pitch=0.8"
```

Comment:
ditto putting this before new var assignments. also add a comment as to what this is meant to sound like because it's incomphrendable what the ultimate intention is